### PR TITLE
solved task

### PR DESCRIPTION
**Pull request by AVykhovanok**

### CODE REVIEW

File: src/main/java/core/basesyntax/service/RegistrationServiceImpl.java [hunk 1]
```diff
@@ -2,13 +2,54 @@
...
+    private boolean isUserLoginCorrect(User user) {
+        return user.getLogin().length() >= 6;
+    }
+
+    private boolean isUserPasswordCorrect(User user) {
+        return user.getPassword().length() >= 6;
+    }
+
+    private boolean isUserAgeCorrect(User user) {
+        return user.getAge() >= 18;
```
Comment (olekskov) [inline src/main/java/core/basesyntax/service/RegistrationServiceImpl.java:53]: please extract magic numbers to constants with proper naming

File: src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    void registerUserAgeException() {
+        Exception exception = Assertions.assertThrows(ValidationException.class,
+                () -> underTest.register(userWrongAge));
+        String expectedMessage = String.format(
+                USER_AGE_EXCEPTION, userWrongAge.getAge());
+        String actualMessage = exception.getMessage();
+        Assertions.assertTrue(actualMessage.contains(expectedMessage));
+    }
+
+    private static void preparedStorage() {
```
Comment (olekskov) [inline src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java:87]:
```suggestion
    private static void prepareStorage() {
```

File: src/main/java/core/basesyntax/service/RegistrationServiceImpl.java [hunk 3]
```diff
@@ -2,13 +2,54 @@
...
+    private boolean isUserExist(User user) {
+        return storageDao.get(user.getLogin()) != null;
+    }
+
+    private boolean isUserLoginCorrect(User user) {
+        return user.getLogin().length() >= 6;
+    }
+
+    private boolean isUserPasswordCorrect(User user) {
+        return user.getPassword().length() >= 6;
```
Comment (olekskov) [inline src/main/java/core/basesyntax/service/RegistrationServiceImpl.java:49]: what if password is null?

File: src/main/java/core/basesyntax/service/RegistrationServiceImpl.java (hunk 1 above)
Comment (olekskov) [inline src/main/java/core/basesyntax/service/RegistrationServiceImpl.java:53]: what if age is null?

**Review by olekskov — CHANGES_REQUESTED** [review]

**Review by uliana-andrushchuk — APPROVED** [review]

**Pull request closed without merging**